patches/chrome-browser-ui-browser.h.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/ui/browser.h b/chrome/browser/ui/browser.h
+index 1633ee2416ae5385a7ad1cd7fdefdad971a427e3..ffa4e457d22055bde4c983a54222c92598bc194d 100644
+--- a/chrome/browser/ui/browser.h
++++ b/chrome/browser/ui/browser.h
+@@ -597,6 +597,7 @@ class Browser : public TabStripModelObserver,
+   friend class BrowserTest;
+   friend class FullscreenControllerInteractiveTest;
+   friend class FullscreenControllerTest;
++  friend class BookmarkPrefsService;
```

Comment:
Most patches should be done as defines now to make them extensible. The convention would be `BRAVE_BROWSER_H` and placed at the end of `public:`. To add `private` entries (like this one) just use `private:` inside the define

Reply:
Done. 👍